.mega-linter.yml

```diff
@@ -25,22 +25,22 @@ DISABLE_ERRORS_LINTERS:
   - REPOSITORY_SEMGREP
   - SPELL_VALE
 PRINT_ALL_FILES: false
-FILTER_REGEX_EXCLUDE: '(\.automation/test|\.automation/generated|\.github/workflows|docs/javascripts|docs/overrides|docs/json-schemas|flavors|clj-kondo|TEMPLATES)'
+FILTER_REGEX_EXCLUDE: '(\.automation/test|\.automation/generated|\.venv|\.github/workflows|docs/javascripts|docs/overrides|docs/json-schemas|flavors|clj-kondo|TEMPLATES)'
 JSON_JSONLINT_FILTER_REGEX_EXCLUDE: '(\.vscode/)'
 YAML_YAMLLINT_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml)'
 YAML_PRETTIER_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml|mkdocs\.yml)'
 YAML_V8R_FILTER_REGEX_EXCLUDE: '(descriptors|templates/\.mega-linter\.yml|\.codecov\.yml)'
 BASH_FILTER_REGEX_EXCLUDE: "(lib)"
 MARKDOWN_FILTER_REGEX_EXCLUDE: '(license\.md|docs/licenses)'
 MARKDOWN_MARKDOWN_LINK_CHECK_DISABLE_ERRORS: true
-SPELL_MISSPELL_FILTER_REGEX_EXCLUDE: '(\.automation/generated|docs/descriptors|used-by-stats|docs/used-by-stats\.md|.github/linters/valestyles)'
+SPELL_MISSPELL_FILTER_REGEX_EXCLUDE: '(\.automation/generated|\.venv|docs/descriptors|used-by-stats|docs/used-by-stats\.md|.github/linters/valestyles)'
 SPELL_VALE_FILE_EXTENSIONS:
   - .md
 SPELL_FILTER_REGEX_EXCLUDE: (valestyles|\.github|docs/licenses)
 DOCKERFILE_HADOLINT_ARGUMENTS: "--ignore DL3003 --ignore DL3007 --ignore DL3013 --ignore DL3016 --ignore DL3018 --ignore DL3028 --ignore DL3059 --ignore DL4001 --ignore DL4006 --ignore SC2015 --ignore SC2016 --ignore SC2039 --ignore SC2086 --ignore SC1091 --ignore SC3046"
 REPOSITORY_TRIVY_ARGUMENTS:
   - "--skip-dirs"
-  - ".automation/test"
+  - ".automation/test .venv"
```

Comment:
@ThomasSanson here you forgot to go in a new line, so you messed with trivy arguments, that's what explains all these errors :)

Reply:
It also seems that --skip-dirs must be written for each dir
https://aquasecurity.github.io/trivy/v0.42/docs/configuration/skipping/#skip-directories

Reply:
Ah okay, I just understood. I was so focused on the single error from Trivy that I ended up getting confused in the process. In fact, I was under the impression that it was normal to detect errors in the .automation/test directory, like for example:

```bash
  .automation/test/docker/bad/Dockerfile (dockerfile)
  ===================================================
  Tests: 20 (SUCCESSES: 17, FAILURES: 3, EXCEPTIONS: 0)
  Failures: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 0, CRITICAL: 2)
```

Thank you for your help ! And also for the solution !

Reply:
Done